Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8
+FROM openjdk:8-jdk
```

Comment:
Why is the jdk needed?

Reply:
The non-jdk version was not working (broken pipe when accessing Docker Hub). The JDK version builds successfully. Maybe it was temporary, but until today it doesn't work. Could you test in your machine?

Reply:
What about `-jre`?

Reply:
According to https://hub.docker.com/r/library/openjdk/tags/, this tag doesn't exist

Reply:
I just provisioned a new vm, installed docker, and it built just fine.
The top one is the new kayenta image and the bottom one is the `openjdk:8` image.

```
duftler$ sudo docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
<none>              <none>              e9eece7d9b97        8 seconds ago       900.3 MB
openjdk             8                   c3e386dcd062        4 days ago          726.1 MB
```